packages/vite-plugin-svelte/src/plugins/load-compiled-css.js

```diff
@@ -37,6 +54,8 @@ export function loadCompiledCss(api) {
 					}
 					css.moduleType = 'css';
 					return css;
+				} else {
+					log.warn(`failed to load virtual css module ${id}`, undefined, 'load');
```

Comment:
with this fix, this should not happen anymore, but we want to know if it does because that would be another issue.